Mbarak-Mbigo/cp2_bucketlist
# Ft create user resource

#### What does this PR do?
Add User login resource
#### Description of Task to be completed?
Add User login resource
#### How should this be manually tested?
run the following `./manage.py test`
Expected results: `All tests for user login should pass`
#### Any background context you want to provide?
For this functionality to work and all areas tested to pass, User registration resource must be fully functional
